packages/client/src/components/ToolItem/ToolItem.component.js

```diff
@@ -120,4 +111,5 @@ ToolItem.defaultProps = {
   groupSizeMin: null,
   pitch: null,
   picture: null,
+  categories: null,
```

Comment:
If the prop type is arrayOf, make the default an empty list `[]` instead.
Else you'll get null pointer error if someone doesn't pass it in because your code doesn't check for null (nor should it).